packages/client/src/views/Home.vue

```diff
@@ -0,0 +1,7 @@
+i<script>
```

Comment:
random `i`.
you can remove the whole `<script>` tag as well.

Reply:
Random `i` is a vim artifact, sometimes you will also see random Js and random Ks 😆